### ISSUE TITLE
Typo in interface.d.ts

### ISSUE THREAD

**Issue by nzacca** [stalniy/bdd-lazy-var#51]
Should this be `global` instead of `gloval`?

https://github.com/stalniy/bdd-lazy-var/blob/master/interface.d.ts#L16

Same for line 17 as well.

**Comment by nzacca** [comment]
Also receiving the following errors:

```
TS2339: Property 'expect' does not exist on type 'Global'.
TS2339: Property 'it' does not exist on type 'Global'.
```

version: 2.4.2

**Comment by stalniy** [comment]
Hi @nzacca 

Thanks for the issue and sorry for the typos. I reverted interface.d.ts until I find the proper way to solve #49 

Fix is available in 2.4.3

**Comment by nzacca** [comment]
Hi @stalniy 

Sorry to trouble you again. It seems there are still some residual types pointing to the types you reverted from interface.d.ts:

```
const is: typeof lv.is;
const its: typeof lv.its;
const xits: typeof lv.xits;
const fits: typeof lv.fits;
```
https://github.com/stalniy/bdd-lazy-var/blob/master/index.d.ts#L5

**Comment by stalniy** [comment]
Ok :) published 2.4.4. Could you please check whether all good?

**Comment by nzacca** [comment]
All good. Thank you very much!
